packages/omd/omd.0.5/opam

```diff
@@ -26,6 +26,6 @@ markdown features."""
 flags: light-uninstall
 extra-files: ["omd.install" "md5=0395736894a46f718a77f59ec6fbf1fd"]
 url {
-  src: "http://pw374.github.io/distrib/omd/omd-0.5.tar.gz"
-  checksum: "md5=d343deacc270674828260b0fdc55f74c"
+  src: "https://github.com/ocaml/omd/archive/refs/tags/0.5.tar.gz"
+  checksum: "sha256=fc462bf7b8f896f924fef610da436d9152168676adca2d606233101eb67e9432"
```

Comment:
```suggestion
  checksum: [
    "sha256=fc462bf7b8f896f924fef610da436d9152168676adca2d606233101eb67e9432"
    "md5=d343deacc270674828260b0fdc55f74c"
  ]
```

Reply:
if the archives are not the same, do we still want the md5 hash?